project/Student.cs

```diff
@@ -141,5 +141,23 @@ public void RemoveAssignmentSubmission(IAssignment assignment)
             assignment.SubmissionDate = null;
             _assignments.Remove(assignment);
         }
+        public void EditAssignmentSubmission(IAssignment assignment)
```

Comment:
I think there would need to be more checks on the children of the IAssignment interface, but I guess for the sake of completing the first attempt it's okay

Reply:
Okay